sdk/trace/src/main/java/io/opentelemetry/sdk/trace/AttributesMap.java

```diff
@@ -67,11 +67,11 @@ public boolean isEmpty() {
 
   @SuppressWarnings({"rawtypes", "unchecked"})
   @Override
-  public void forEach(BiConsumer<AttributeKey<?>, Object> consumer) {
+  public void forEach(BiConsumer<? super AttributeKey<?>, ? super Object> consumer) {
     for (Map.Entry<AttributeKey<?>, Object> entry : data.entrySet()) {
       AttributeKey key = entry.getKey();
```

Comment:
Why do we use a raw type here?

```suggestion
      AttributeKey<?> key = entry.getKey();
```